docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+
+   ```
+   Bundle bundle = new Bundle();
+   bundle.AddItem("key", "value");
+   ```
+
+2. Use the `Publish()` method to publish the event:
+
+   ```
+   ApplicationEventManager.Publish("event.org.tizen.example.AppEventTestApp.AppEvent", bundle);
+   ```
+
+<a name="manage"></a>
+## Subscribing to an Event
+
+To subscribe to a predefined system event or user-defined event:
+
+1. Add an event handler.
+
+   One event can have multiple event handlers, and one handler can be registered multiple times.
+
+   - Add an event handler for a system event:
+
+     ```
+     void OnReceived(object sender, EventManagerEventArgs e)
+     {
+         LogUtils.Write(LogUtils.DEBUG, LOG_TAG, "On Received : " + e.Name);
+     }
+
+     /* Register the event handler */
+     EventReceiver receiver = new EventReceiver(SystemEvents.BatteryChargerStatus.EventName);
+     receiver.Received += OnReceived;
+     ```
+
+   - Add an event handler for a user-defined event:
+
+     When defining an event name for a user event such as `event.org.tizen.senderapp.user_event`, the name format is `event.{sender appid}.{user-defined name}`. The `{user-defined name}` must:
+
+     - Contain only the ASCII characters "[A-Z][a-z][0-9]_" and not begin with a digit.
+     - Not contain a '.' (period) character.
+     - Not exceed the maximum name length (127 bytes).
+     - Be at least 1 byte in length.
+
+     ```
+     EventReceiver receiver = new EventReceiver("event.org.tizen.example.AppEventTestApp.AppEvent");
+     receiver.Received += OnReceived;
+     ```
+
+2. Remove the event handler when no longer needed.
+
+   A registered handler can be removed when the application is running. All the registered handlers can be removed when the application is terminated:
+
+   ```
+   receiver.Received -= OnReceived;
+   ```
+
+<a name="launch"></a>
+## Managing Launch-On-Events
+
+To register an interest in a Launch-On-Event, define the `http://tizen.org/appcontrol/operation/launch_on_event` operation in the `tizen-manifest.xml` file.
+
+> **Note**
+>
+> Only service applications can register and receive Launch-On-Events.
+>
+> The Launch-On-Event operation cannot be requested using the `AppControl.SendLaunchRequest()` method, unlike other application control operations.
+
+The following table shows the system events that support Launch-On-Event:
+
+**Table: System events supporting Launch-On-Event**
+
+| Name                                     | Condition                                |
+|------------------------------------------|------------------------------------------|
+| `SystemEvents.BatteryChargerStatus.EventName` | When the charger state is `SystemEvents.BatteryChargerStatus.StatusValueConnected`. |
+| `SystemEvents.UsbStatus.EventName`         | When the USB state is `SystemEvents.UsbStatus.StatusValueConnected`.     |
+| `SystemEvents.EarjackStatus.EventName`     | When the earjack status is `SystemEvents.EarjackStatus.StatusValueConnected`. |
```

Comment:
Please make this small change to be consistent with the other sentence in the table:

When the earjack state is `SystemEvents.EarjackStatus.StatusValueConnected`.